templates/flutter/lib/src/realtime_mixin.dart.twig

```diff
@@ -76,6 +76,23 @@ mixin RealtimeMixin {
             }
             break;
         }
+      }, onDone: () {
+        for (var list in _channels.values) {
+          for (var stream in list) {
+            stream.close();
+          }
+        }
+        _closeConnection();
```

Comment:
Is it a problem if the subscription isn't removed from `_channels`? If a new subscription is created, the old subscription could also be used, resulting in an error like:

```
Error: Bad state: Cannot add new events after calling close
```

Reply:
I reset channel once done, this should probably solve this.